client/src/pages/_app.tsx

```diff
@@ -94,15 +95,17 @@ const MyApp: React.FC<AppProps> = ({ Component, pageProps }: AppProps) => {
           > */}
         {/* @ts-ignore: https://github.com/artsy/fresnel/issues/281 */}
         <MediaContextProvider>
-          <MetaIcons />
+          <PlausibleProvider domain="fora-frontend.vercel.app">
```

Comment:
If the application is deployed outside of Vercel, this won't work as it's hardcoded. I suggest moving this value to an environmental variable like `NEXT_PUBLIC_PLAUSIBLE_DOMAIN=fora-frontend.vercel.app`  and updating the variable when needed.